test/e2e/serial/tests/workload_unschedulable.go

```diff
@@ -229,4 +229,156 @@ var _ = Describe("[serial][disruptive][scheduler] numaresources workload unsched
 			}
 		})
 	})
+
+	Context("with at least one node", func() {
+		It("[test_id:47616][tier2][tmscope:pod] pod with two containers each on one numa zone can NOT be scheduled", func() {
+
+			// Requirements:
+			// Need at least this nodes
+			neededNodes := 1
+			// with at least this number of numa zones
+			requiredNUMAZones := 2
+			// and with this policy
+			tmPolicy := nrtv1alpha1.SingleNUMANodePodLevel
+
+			// filter by policy
+			nrtCandidates := e2enrt.FilterTopologyManagerPolicy(nrtList.Items, tmPolicy)
+			if len(nrtCandidates) < neededNodes {
+				Skip(fmt.Sprintf("not enough nodes with policy %q - found %d", string(tmPolicy), len(nrtCandidates)))
+			}
+
+			// Filter by number of numa zones
+			By(fmt.Sprintf("filtering available nodes with at least %d NUMA zones", requiredNUMAZones))
+			nrtCandidates = e2enrt.FilterZoneCountEqual(nrtCandidates, requiredNUMAZones)
+			if len(nrtCandidates) < neededNodes {
+				Skip(fmt.Sprintf("not enough nodes with 2 NUMA Zones: found %d, needed %d", len(nrtCandidates), neededNodes))
+			}
+
+			// filter by resources on each numa zone
+			requiredResCnt1 := corev1.ResourceList{
+				corev1.ResourceCPU:    resource.MustParse("4"),
+				corev1.ResourceMemory: resource.MustParse("4Gi"),
+			}
+
+			requiredResCnt2 := corev1.ResourceList{
+				corev1.ResourceCPU:    resource.MustParse("5"),
+				corev1.ResourceMemory: resource.MustParse("5Gi"),
+			}
+
+			By("filtering available nodes with allocatable resources on at least one NUMA zone that can match request")
+			nrtCandidates = e2enrt.FilterNRTsEachRequestOnADifferentZone(nrtCandidates, requiredResCnt1, requiredResCnt2)
```

Comment:
I'd prefer this helper to be moved in `internal/` (to be decided where) and gain unit test coverage. To move forward with this PR, we can (and probably should) have a super simple local helper which checks both the combinations: cnt1/zone1 and cont2/zone2 vs cnt1/zone2 and cnt2/zone1.